### PR TITLE
webui: availability map update

### PR DESCRIPTION
**Pull request by crcro**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

- smaller devices box on new view due to shorthostname usage
- fixed several css classes
- added on/off webui global setting for device groups (issue: #4275)
- renamed all vars to match var_name style (varName style used)

### CODE REVIEW

File: html/includes/common/availability-map.inc.php [hunk 1]
```diff
@@ -220,6 +235,32 @@
             '</select>
         </div>
         <div class="page-availability-title-right">';
+
+        $sql = 'SELECT `G`.`id`, `G`.`name` FROM `device_groups` AS `G`';
+        $dev_groups = dbFetchRows($sql);
```
Comment (murrant) [inline html/includes/common/availability-map.inc.php]: Maybe put the sql query inside the check for availability_map_use_device_groups check so it isn't executed every time.  Unless you need it.

Comment (crcro) [inline html/includes/common/availability-map.inc.php]: you are right, moved inside if.

**Comment by breeman1** [comment]
Hi @crcro,

Thanks for the update, however this update causes long hostnames to move outside the "new smaller boxes. The older bigger boxes were more accommodating of long hostname which was desirable and neater in our case. Isn't there perhaps a "word wrap" like feature that can help contain the hostnames inside the boxes? See below;
snap3

**Comment by breeman1** [comment]
The feature to be able to choose by device groups is welcomed. However as a suggestion I think that option should not change the default behavior of showing all devices even though an option to select/filter by device groups has been enabled. So ideally, the availability-map screen should show all devices by default but have the drop down menu available to filter by device groups should one have that option enabled in the global config.

**Comment by crcro** [comment]
@breeman1 i think that PR#4334 will fix your issues

**Comment by breeman1** [comment]
Yes it did. Thank you.

I had to change the `width:163px;` to `width:168px;` in the `html/css/styles.css` file for one of the boxes; the last one in the picture of my previous post, to fit the longest hostname in our test install.

**Pull request merged**